uaclient/apt.py

```diff
@@ -754,8 +771,10 @@ def _get_apt_config():
 
 
 def get_apt_config_keys(base_key):
-    apt_cfg = _get_apt_config()
-    return apt_cfg.list(base_key)
+    with PreserveAptCfg(_get_apt_config) as apt_cfg:
```

Comment:
I don't understand why we need the ctx change here if we want the current apt config... but, if it's needed here, wouldn't it be needed in `get_apt_config_values` too?